### DISCUSSION TITLE
Does selecting a Group check limit the run to ONLY those Checks/Extras in the Group?

### DISCUSSION THREAD

**Discussion by edschreibman** [prowler-cloud/prowler#1463] [Q&A] [answered]
If I run prowler with a -g for example:   ./prowler -g hipaa
Does that mean ONLY the Groups/Checks in that group are used?

for HIPAA as an example:
[https://github.com/prowler-cloud/prowler/blob/master/groups/group10_hipaa](url)
I assume Prowler is ONLY using the checks/extras as listed in the line that starts with 
 **GROUP_CHECKS[10]=**'check12,check113...'

If I run it with no flag are **all** checks and no "extras" run?
If I run with -g extras are **ALL** extras and **ALL** checks run? (except experimental) ?

**Comment by jfagoagas** [comment, marked as answer]
Hi @edschreibman, first of all thank you for using Prowler.

- If you run `/prowler -g hipaa` only the checks included [here (group10_hipaa)](https://github.com/prowler-cloud/prowler/blob/master/groups/group10_hipaa) will run. You're right, only the checks included in `GROUP_CHECKS[10]`.
- If you run Prowler without flags like `./prowler` the 243 checks will be executed, both **checks** and **extras**.
- If you run `./prowler -g extras` only the checks included in the **extras** groups will be executed https://github.com/prowler-cloud/prowler/blob/a769bb86d3afa86277ce42e8354ecdeb568aba6a/groups/group7_extras#L17

Thanks again!
